CIP-1694/README.md

```diff
@@ -420,22 +432,128 @@ However, this choice also limits the implementation effort for wallet providers
 
 Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
 
+### The purpose of the DReps
+
+Nothing in this proposal limits the SPOs from becoming DReps.
+Why do we have DReps at all?
+SPOs are chosen purely for block production.
+Voters can choose to delegate their vote to DReps without needing to consider whether they are
+also a good block producer.
+
+### AVST (action voting stake threshold)
+
+The AVST exists to ensure that the votes of the DReps are meaningful.
+For example, if only 10 Lovelace were properly delegated to DReps, and 9 Lovelace "vote yes" to
+a governance action, we would doubt the legitimacy of the result since there are
+billions of ADA in circulation.
+A high enough AVST legitimizes the representation.
+
+### Ratification Requirements Table
+
+The requirements in the [ratification requirement table](#requirements) are explained here.
+Most of the governance actions have the same kind of requirements:
+the constitutional committee must reach quorum and the DReps must reach a sufficient number of
+'yes' votes, except in the situation where the AVST is too low, in which case the SPOs vote in
+place of the DReps.
+This includes these actions:
+* New Committee/quorum (normal state)
+* Update to the Constitution
+* Protocol parameter changes
+* Treasury withdrawal
+
+#### Motion of no-confidence
+
+A motion of no-confidence represents the lack of confidence by the Cardano community in the
+current Constitutional Committee, and hence the Constitutional Committee should have no
+say whatsoever in this governance action.
+In this situation, the SPOs and the DReps are left to represent the will of the community.
+To prevent an under-representation of the DReps, the AVST (active voting stake threshold)
+becomes a requirement for ratification.
+
+#### New Committee/quorum (state of no-confidence)
+
+Similar to the motion of no-confidence, electing a constitutional committee
+depends on both the SPOs and the DReps to represent the will of the community.
+To prevent an under-representation of the DReps, the AVST (active voting stake threshold)
+becomes a requirement for ratification.
+
+#### Hard-Fork initiation
+
+Regardless of any governance mechanism, SPO participation is needed for any hard fork since it is
+they who must upgrade their software.
+For this reason, we make their cooperation explicit in the hard fork initiation governance action,
+by always requiring their vote.
+The Constitutional Committee also votes, signaling the constitutionality of a hard fork.
+The DReps also vote, to represent the will of every stake holder, though the AVST is not required.
```

Comment:
A DRep vote without AVST seems pretty meaningless tbh. I think we might as well just say that only the SPOs vote.

Reply:
Doesn't the current setup mean DReps still have veto power over HFs?

Reply:
Right, they do, and with no AVST that's arguably bad! That means that if there is low DRep engagement then whatever arbitrarily small group of DReps do vote can veto a HF. Maybe that's desirable. Regardless I think this indicates that the logic of which properties you get for different vote and thresholds is _quite complex_ and could stand to be spelled out extremely explicitly.

Reply:
Seems a bit redundant IMO, since, in practice SPOs are the only ones who can, physically, initiate a HF,by the majority of them upgrading their nodes, if I understand it correctly.